### ISSUE TITLE
[FEATURE] Download size preview?

### ISSUE THREAD

**Issue by maxappel** [robvdpol/RaceControl#134]
Would it be possible to add a feature to show how large a download will be?

**Comment by robvdpol** [comment]
@SuRGeoNix thoughts?

**Comment by SuRGeoNix** [comment]
Yep, I will have a look to included in the next update (it will take some time though for the next update)
